packages/strapi-plugin-graphql/services/GraphQL.js

```diff
@@ -28,36 +28,57 @@ module.exports = {
       type === 'Int' ||
       type === 'Float' ||
       type === 'String' ||
-      type === 'Boolean'
+      type === 'Boolean' ||
+      type === 'DateTime'
     );
   },
 
+  /**
+   * Checks if the field is of type enum
+   */
+  isEnumType: (_type) => {
+    return _.startsWith(_type, 'ENUM_');
```

Comment:
hey @kamalbennani 
Actually, an enum can start with the `ENUM_` by default, but it can also be overrided to a custom name.
What about detect if it's an `EnumTypeDefinition` instead?

Reply:
I'll go with your solution